crates/gitbutler-core/src/virtual_branches/branch/hunk.rs

```diff
@@ -2,14 +2,17 @@ use std::{fmt::Display, ops::RangeInclusive, str::FromStr};
 
 use anyhow::{anyhow, Context, Result};
 
-use crate::git::diff;
+use crate::{git::diff, id::Id};
+
+use super::Branch;
 
 #[derive(Debug, Eq, Clone)]
 pub struct Hunk {
     pub hash: Option<String>,
     pub timestamp_ms: Option<u128>,
     pub start: u32,
     pub end: u32,
+    pub locked_to: Option<Vec<Id<Branch>>>,
```

Comment:
Consider dropping the Option here - since you have a Vec, it can be empty

Reply:
Ah, thanks, that was added by mistake.